scripts/src/create.ts

```diff
@@ -18,17 +18,17 @@ import "./models/orders";
 import "./models/users";
 
 getConfig();
+let config;
```

Comment:
this is confusing, change this to be a specific config as a line above you have a different type of config.
also, give it a type annotation here so it will be clear.

Reply:
Cool